examples/api/filled_step.py

```diff
@@ -3,7 +3,7 @@
 Hatch-filled histograms
 =========================
 
-This example showcases the hatching capabilities of matplotlib by plotting
+Hatching capabilities of matplotlib by plotting
```

Comment:
Sentence fragment doesn't work anymore.

Reply:
How about "Hatching capabilities for plotting histograms"?